.circleci/config.yml

```
@@ -51,8 +51,11 @@ commands:
     steps:
       - checkout
       - run:
-          name: Snapshot database and run migrations
-          command: bin/do-exclusively --job-name ${CIRCLE_JOB} bin/rds-snapshot-app-db $APP_ENVIRONMENT && bin/ecs-run-app-migrations-container config/app-migrations.container-definition.json ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/app-migrations:git-${CIRCLE_SHA1} $APP_ENVIRONMENT
+          name: Snapshot database
+          command: bin/do-exclusively --job-name ${CIRCLE_JOB} bin/rds-snapshot-app-db $APP_ENVIRONMENT
+      - run:
+          name: Run migrations
+          command: bin/do-exclusively --job-name ${CIRCLE_JOB} bin/ecs-run-app-migrations-container config/app-migrations.container-definition.json ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/app-migrations:git-${CIRCLE_SHA1} $APP_ENVIRONMENT
```

I think this is a good change but is there an issue with spitting it into two steps?

---

I was looking into that.  I think I confirmed that `time` doesn't swallow error messages.  The error we were seeing was really a failure of ECS to obtain the correct network settings to set up the task for migration.  However, its probably worth some more investigation into the snapshot code to ensure that when it fails we don't continue any processes.

---

The new check looks at the snapshot description and compares the status text.